lib/slack-ruby-bot/rspec/support/slack-ruby-bot/respond_with_slack_message.rb

```diff
@@ -13,8 +13,9 @@
 
     allow(Giphy).to receive(:random) if defined?(Giphy)
 
-    expect(client).to receive(:message).with(channel: channel, text: expected)
+    allow(client).to receive(:message)
     message_command.call(client, Hashie::Mash.new(text: message, channel: channel, user: user))
+    expect(client).to have_received(:message).with(channel: channel, text: expected).once
```

Comment:
I believe this changes behavior, ie. it allows the client to receive multiple messages that aren't the one expected plus the one expected. I think we don't want that, we want expectations to be "precise".